packages/react-lowcode/src/codegen/ast/node.ts

```diff
@@ -20,4 +20,12 @@ export const isOpeningOrSelfClosingElementWithName = (node: ts.Node, name: strin
             }
         }
     }
+}
+
+export const isUseQueryHook = (node: ts.Node, name: string): boolean | undefined => {
+  if(ts.isCallExpression(node)) {
+    if(node.expression.getFullText().indexOf(name) > 0) {

```

Comment:
what is usual value of `node.expression.getFullText()`?
think about `>=` or '=='